src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -202,9 +202,7 @@
                     <TabItem Header="{x:Static p:Resources.PreferencesViewGeneralTab}"
                              Style="{StaticResource LeftTab}">
 
-                        <ScrollViewer VerticalScrollBarVisibility="Auto" 
-                                              Height="Auto">
-                            <!--This Grid contains controls for selecting the Language-->
```

Comment:
Would you keep the comments here?

Reply:
> Would you keep the comments here?

My bad since I reorganized various controls here